horton/meanfield/test/common.py

```diff
@@ -546,3 +547,123 @@ def check_vanadium_sc_hf(scf_solver):
 
     # SCF test
     check_solve(ham, scf_solver, occ_model, lf, olp, kin, na, exp_alpha)
+
+
+@log.with_level(log.high)
+def check_water_cs_m05(scf_solver):
+    """Try to converge the SCF for the water molecule with the M05 functional.
+
+    Parameters
+    ----------
+    scf_solver : one of the SCFSolver types in HORTON
+                 A configured SCF solver that must be tested.
+    """
+    fn_fchk = context.get_fn('test/water_m05_321g.fchk')
+    mol = IOData.from_file(fn_fchk)
+    grid = BeckeMolGrid(mol.coordinates, mol.numbers, mol.pseudo_numbers, 'fine',
+                        random_rotate=False)
```

Comment:
Ok. For testing, turning it of is better. The accuracy is a little less but the numbers become more reproducible.